.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+labels: bug
+---
+
+<!--
+Thanks for reporting a bug report to CLIMA, the Climate Machine!
+
+Please fill in the information below.
+
+If you would like to contact us, we are also available on Slack (What is the best contact method)?
+-->
+
+
+# Description
+
+A clear and concise description of what the bug is.
+
+## Steps to Reproduce
+
+Steps to reproduce the behavior
```

Comment:
Suggestions on making this clearer 
1. State of repository (e.g. `branchname`, `commit ID`)
    - For bugs maybe even one step further (e.g. create a new branch off master e.g. `bug-<something>` that allows developers to isolate the problem, has the drawback of potentially polluting the branch list & requires issue handlers to manually delete bug-demo-branches once bugs are resolved...)
2. Exact sequence of commands run (either in REPL or terminal) to reproduce this issue
3. Expected wall-clock and simulation-time for issue/bug to reveal itself (e.g. in the case of problems with physics / set-ups that require longer time to solution)